frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedTab.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import 'leaflet';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import 'leaflet/dist/leaflet.css';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { IParcel, IBuilding } from 'actions/parcelsActions';
+import { render } from '@testing-library/react';
+import { mockParcel, mockBuilding } from './InfoContent.test';
+import AssociatedParcelsList from './AssociatedParcelsList';
+import { Label } from 'components/common/Label';
+import { FaPlusSquare } from 'react-icons/fa';
+import AssociatedBuildingsList from './AssociatedBuildingsList';
+
+Enzyme.configure({ adapter: new Adapter() });
+const history = createMemoryHistory();
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({});
+
+const addAssociatedBuildingLink = (
+  <>
+    <FaPlusSquare color="#1a5a96" className="mr-1" />
+    <Label>Add a new Building</Label>
```

Comment:
for the purposes of the test this doesn't actually need to be a link

Reply:
(at least I don't think so?)

Reply:
there is an argument for it being the same component. if someone was to break the actual addAssociatedBuildingLink component in some way, because you have created a proxy here the test would still pass.

Reply:
Ah, I see what you mean. None of the tests do anything with the link at the moment though, it's just needed to render the component

Reply:
yeah, I think with the current implementation nothing needs to change.